deployment/secure_api_server.py

```
@@ -555,6 +593,171 @@
+        except ValueError as e:
+            response_time = time.time() - start_time
+            update_metrics(response_time, success=False, error_type='validation_error')
+            return jsonify({'error': str(e)}), 400
+
+        sanitized_text, warnings = input_sanitizer.sanitize_text(text, "emotion")
+        try:
+            service = get_emotion_service()
+        except (ImportError, ValueError) as e:
+            response_time = time.time() - start_time
+            update_metrics(response_time, success=False, error_type='provider_error')
+            return jsonify({'error': f'Emotion provider misconfiguration: {str(e)}'}), 503
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)